tests/encodeinput.cpp

```diff
@@ -167,40 +167,47 @@ EncodeOutput::~EncodeOutput()
         fclose(m_fp);
 }
 
-EncodeOutput* EncodeOutput::create(const char* outputFileName, int width , int height)
+EncodeOutput* EncodeOutput::create(const char* outputFileName, int width,
+                                   int height, const char* codecName)
 {
-    EncodeOutput * output = NULL;
-    if(outputFileName==NULL)
+    EncodeOutput* output = NULL;
+    if (outputFileName == NULL)
         return NULL;
-    const char *ext = strrchr(outputFileName,'.');
-    if(ext==NULL)
+    const char* ext = strrchr(outputFileName, '.');
+    std::string codec(""); // needed for strcasecmp to work
```

Comment:
We already have a global variable here https://github.com/01org/libyami-utils/blob/master/tests/encodehelp.h#L28, But it default to NULL. I mean, if you set this default to “”
It will make code more cleaner.Just a suggestion ;)